README.md

```diff
@@ -1,13 +1,25 @@
 # Student template
-
+[![Coverage Status](https://coveralls.io/repos/github/erictelkkala/COMP.SE.200-2024-2025-1/badge.svg)](https://coveralls.io/github/erictelkkala/COMP.SE.200-2024-2025-1)
 ## Purpose of this repository
 
-This is a project template for students participating in Software Testing course
-at Tampere University.
+This is projects implements unit tests for files in the upstream repository.
+
+## Running tests locally
+Tests can be run locally with:
+```shell
+npm t
+```
```

Comment:
https://docs.npmjs.com/cli/v10/commands/npm-test

> 
> npm test [-- <args>]
> 
> aliases: tst, t

So basically the `t` is just an alias for `test`.

Maybe it should be changed to be `test` instead of `t` for more clarity?

Reply:
`t` Is okay as it runs the tests.